doc/california_law/California_law.md

```diff
@@ -0,0 +1,236 @@
+# California Law Design #
+
+##  1. <a name='TableofContent'></a>Table of Content
```

Comment:
can we use a more descriptive name? "Default Credential Management for California Law XXX Conformance"?